apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.ts

```diff
@@ -32,7 +42,64 @@ router.get(
       if (row.sync_warnings) row.sync_warnings_ansified = ansiUp.ansi_to_html(row.sync_warnings);
       return row;
     });
-    res.send(InstructorAssessmentQuestions({ resLocals: res.locals, questions }));
+    const assessmentPath = path.join(
+      res.locals.course.path,
+      'courseInstances',
+      res.locals.course_instance.short_name,
+      'assessments',
+      res.locals.assessment.tid,
+      'infoAssessment.json',
+    );
+
+    const assessmentPathExists = await fs.pathExists(assessmentPath);
+
+    let origHash = '';
+    if (assessmentPathExists) {
+      origHash = sha256(b64EncodeUnicode(await fs.readFile(assessmentPath, 'utf8'))).toString();
+    }
+
+    const assessmentQuestionEditorEnabled = await features.enabledFromLocals(
+      'assessment-questions-editor',
+      res.locals,
+    );
+
+    res.send(
+      InstructorAssessmentQuestions({
+        resLocals: res.locals,
+        questions,
+        origHash,
+        assessmentQuestionEditorEnabled,
+      }),
+    );
+  }),
+);
+
+router.get(
+  '/findqid',
+  asyncHandler(async (req, res) => {
+    const courseInstances = await selectCourseInstancesWithStaffAccess({
+      course_id: res.locals.course.id,
+      user_id: res.locals.user.user_id,
+      authn_user_id: res.locals.authn_user.user_id,
+      is_administrator: res.locals.is_administrator,
+      authn_is_administrator: res.locals.authz_data.authn_is_administrator,
+    });
+
+    const questions: QuestionsPageDataAnsified[] = await selectQuestionsForCourse(
```

Comment:
Declaring a specific type shouldn't be necessary:

```suggestion
    const questions = await selectQuestionsForCourse(
```

Reply:
Removed in [3c6791f](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/3c6791f5b7ef00282d313997acf9af8418da3660).